lib/matplotlib/backends/backend_qt5.py

```diff
@@ -763,18 +766,13 @@ def edit_parameters(self):
             ax = axes[titles.index(item)]
         figureoptions.figure_edit(ax, self)
 
-    def _update_buttons_checked(self):
-        # sync button checkstates to match active mode
-        self._actions['pan'].setChecked(self._active == 'PAN')
-        self._actions['zoom'].setChecked(self._active == 'ZOOM')
-
     def pan(self, *args):
         super().pan(*args)
-        self._update_buttons_checked()
+        self._actions['pan'].setChecked(self._active == 'PAN')
 
     def zoom(self, *args):
         super().zoom(*args)
-        self._update_buttons_checked()
+        self._actions['zoom'].setChecked(self._active == 'ZOOM')
```

Comment:
how would this un-check `self._actions['pan']`? Or does it no longer need to?

Reply:
The QButtonGroup takes care of unchecking any other button (i.e., pan).